LayoutTests/fast/css/background-position-serialize.html

```diff
@@ -99,6 +99,41 @@
 shouldBe("t.style.cssText", "background-position: 20% 20%;");
 shouldBe("t.getAttribute('style')", "background-position: 20% 20%;");
 
+run("t.style.backgroundPositionY = '0%';");
+shouldBe("t.style.backgroundPosition", "20% 0%");
+shouldBe("t.style.backgroundPositionX", "20%");
+shouldBe("t.style.backgroundPositionY", "0%");
+shouldBe("t.style.cssText", "background-position: 20% 0%;");
+shouldBe("t.getAttribute('style')", "background-position: 20% 0%;");
+
+run("t.style.backgroundPositionX = '0%';");
+shouldBe("t.style.backgroundPosition", "0%");
+shouldBe("t.style.backgroundPositionX", "0%");
+shouldBe("t.style.backgroundPositionY", "0%");
+shouldBe("t.style.cssText", "background-position: 0%;");
+shouldBe("t.getAttribute('style')", "background-position: 0%;");
```

Comment:
This looks incorrect:

https://drafts.csswg.org/css-backgrounds/#background-position

> If only one value is specified, the second value is assumed to be [center](https://drafts.csswg.org/css-backgrounds/#valdef-background-position-center).

So this case should serialize as `background-position: 0% 0%;`.

Reply:
Thanks. I knew there was a reason I added that test case!

Reply:
This second flavor of the bug is caused by a different code path, the "if all are skipped serialize the first" rule.